### DISCUSSION TITLE
Сreate one rule for all windows

### DISCUSSION THREAD

**Discussion by Ved-un** [leftwm/leftwm#1165] [Q&A] [answered]
Hello.    
Can you tell me how to create one rule for all windows?     

What should I put in _window_class:_ ?      
```
    window_rules: [
        (window_class: "", spawn_floating: true),
    ],
```

**Comment by Eskaan** [comment, marked as answer]
`window_class` should be a regex, so I think `window_class: ".*"` should work.

**Comment by Ved-un** [comment]
Thanks
